src/core/issues-notes.js

```diff
@@ -239,12 +241,24 @@ function isLight(rgb) {
  * @param {string} title
  * @param {string} repoURL
  */
-function createLabelsGroup(labels, title, repoURL) {
+function createLabelsGroup(conf, labels, title, repoURL) {
   const labelsGroup = labels.map(label => createLabel(label, repoURL));
+  let labelName = labels.map(label => label.name);
+  const totalLabels = labels.length;
+  const reducer = (string, label, index) => {
+    return index === totalLabels - 1
+      ? `${string} and ${label}`
+      : `${string}, ${label}`;
+  };
+  labelName = labelName.reduce(reducer);
```

Comment:
Oh, apologies, it occurred to me that we have a util function you can import for this: 

https://github.com/w3c/respec/blob/develop/src/core/utils.js#L382

After importing it you can just do:

```JS
const labels = joinAnd(labelNames);
```